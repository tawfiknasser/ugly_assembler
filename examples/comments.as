add r1,r2



; i'm comment...

; i'm comment too
add r1,r2