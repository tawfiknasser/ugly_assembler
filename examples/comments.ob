              4  0  
100       .....././///..
101       ...../..../...
102       .....././///..
103       ...../..../...

