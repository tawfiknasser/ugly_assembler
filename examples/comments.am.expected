add r1,r2
add r1,r2